src/legacy/ui/public/agg_types/buckets/date_histogram.js

```diff
@@ -157,8 +157,8 @@ export const dateHistogramBucketAgg = new BucketAggType({
 
         const scaleMetrics = interval.scaled && interval.scale < 1;
         if (scaleMetrics && aggs) {
-          const all = _.every(aggs.bySchemaGroup.metrics, function (agg) {
-            return agg.type && agg.type.isScalable();
+          const all = _.every(aggs.filter(agg => agg.type && agg.type.type === 'metric'), function (agg) {
```

Comment:
i could use `aggs.byTypeType.metric` here but we want to get rid of using indexed array in aggs